sdk/src/main/java/com/mongodb/stitch/android/StitchClient.java

```diff
@@ -775,6 +782,11 @@ public void onErrorResponse(final VolleyError error) {
         return future.getTask();
     }
 
+    private boolean isTokenExpired(final String jwt) {
```

Comment:
[minor] This method may be better suited be a public method within the `DecodedJWT` class. And since these tokens actually are JWTs, we might as well change the type of the tokens (within the `Auth` class)  to be typed as `DecodedJWT`s to better represent their state.